### PR TITLE
Automatically fill the email on the login with the API token provided.

### PR DESCRIPTION
**Pull request by minuscorp**
Fixes https://github.com/fastlane/ci/issues/243

### CODE REVIEW

File: app/features/login/views/login_fastlane_ci.erb [hunk 1]
```diff
@@ -15,7 +15,7 @@
           <hr />
 
           <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
-            <input class="mdl-textfield__input" type="text" name="email" id="email">
```
Comment (KrauseFx) [inline app/features/login/views/login_fastlane_ci.erb]: Is that the only field we have to auto-fill? I think we ask for 2 API tokens, one for the bot, one for the user, right? I guess we could apply this for both

File: app/features/login/login_controller.rb [hunk 2]
```diff
@@ -33,8 +36,15 @@ class LoginController < ControllerBase
 
     # Login with fastlane.ci credentials
     get "#{HOME}/ci_login" do
+
```
Comment (KrauseFx) [inline app/features/login/login_controller.rb]: This space shouldn't be here

File: app/features/login/views/login_fastlane_ci.erb (hunk 1 above)
Comment (minuscorp) [reply app/features/login/views/login_fastlane_ci.erb]: I'm not aware of more than the environment variable with the initial clone token to be used after the onboarding

File: app/features/login/login_controller.rb [hunk 3]
```diff
@@ -33,8 +36,14 @@ class LoginController < ControllerBase
 
     # Login with fastlane.ci credentials
     get "#{HOME}/ci_login" do
+      client = Octokit::Client.new(access_token: FastlaneCI.env.clone_user_api_token)
```
Comment (KrauseFx) [inline app/features/login/login_controller.rb]: I just realized we're accessing Octokit in a controller, is @taquitos ok with this?

**Pull request merged**